features/element-hiding.json

```diff
@@ -1966,6 +1966,23 @@
                     }
                 ]
             },
+            {
+                "domain": "mantan-web.jp",
+                "rules": [
+                    {
+                        "selector": "center > div",
```

Comment:
I can't see a better selector to use here, hopefully by being specific with the `<center><div></div></center>` structure and using a "hide-empty" rule (which will only hide the element if it's considered empty) this is good enough.

Reply:
Works for me